torch/_meta_registrations.py

```diff
@@ -1635,6 +1635,29 @@ def upsample_nearest2d_vec(input, output_size, scale_factors):
         memory_format=mem_format
     )
 
+def check_argmax_argmin(name, self, dim):
```

Comment:
it's helpful to quote the cpp file this came from aten/src/ATen/native/ReduceOps.cpp